libs/stats/docs/pq-cfg.yaml

```diff
@@ -0,0 +1,69 @@
+# Sample configuration file for `odc-stats run` command
+#
+#    odc-stats run --config <this-file|content of this file> ...
+
+# Configure Plugin and Product Attributes
+plugin: pq
+
+# Generic product attributes
+product:
+  name: pc_s2_annual
+  short_name: pc_s2_annual
+  version: 0.0.1
+  product_family: pixel_quality_statistics
+
+  # specific to a site: Africa
+  # (for AU just keep defaults)
+  #collections_site: collections.digitalearthafrica.org
+  collections_site: explorer.digitalearth.africa
+  producer: digitalearthafrica.org
+
+  # Any extra fields to add to .properties section of the STAC/EO3
+  # for each dataset document
+  properties: {}
+
+# Plugin Specific Configuration
+# defaults for PQ are currently this
+#
+# plugin_config:
+#   filters: [[2, 5], [0, 5]]
+#   resampling: nearest
+
+# Other common settings
+
+# input data is public-access
+aws_unsigned: true
+
+# mark output files as public read (default is false)
+s3_public: true
+
+# Completion deadline in seconds, 0 (default) means take
+# as along as it takes
+max_processing_time: 1200
+
+# SQS specific settings
+#  300s, i.e. 5 minutes is max lease at any given time
+#  extended when getting within 30s of expiry
+#  polling every 5 seconds
+job_queue_max_lease: 300
+renew_safety_margin: 30
+future_poll_interval: 5
+
+# TIFF compression settings
+cog_opts:
+  # Default settings for bands
+  compression: deflate
+  zlevel: 9
+  blocksize: 800
+
+  # overview settings
+  ovr_blocksize: 512
+  overview_resampling: average
+  overview_levels: [2, 4, 8, 16, 32]
+
+  # Bands that require special treatment
+  # Applies to Geomedian Product
+  overrides:
+    rgba:
+      compression: webp
+      webp_level: 90
```

Comment:
also delete and stick with defaults

Reply:
I've removed all the redundant configs.  If you're happy, just leave a comment and I'll use that as the final.